src/entities/Proposal/model.ts

```diff
@@ -162,4 +162,27 @@ export default class ProposalModel extends Model<ProposalAttributes> {
 
     return proposals.map(this.parse)
   }
-}
\ No newline at end of file
+
+  static async getPassedProposals(proposal_type: ProposalType) {
```

Comment:
please don't add custom queries if you don't need it for performance reasons.

is preferible to use the old endpoint so we can apply cache strategies if it is required